mkosi.conf.d/20-centos.conf

```diff
@@ -13,3 +13,4 @@ Packages=
         kpatch-build
         kpatch-dnf
         kpatch-runtime
+        systemd-resolved
```

Comment:
This can be moved to 20-centos-fedora.conf

Reply:
I thought with the merge of the centos and fedora configs you'd prefer it this way, but will do.